crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -486,11 +533,28 @@ where
             .collect::<Vec<_>>()
     }
 
+    /// Returns the [EthStateCache] frontend
+    ///
+    /// This will spawn exactly one [EthStateCache] service if this is the first time the cache is
+    /// requested.
+    pub fn eth_cache(&mut self) -> EthStateCache {
```

Comment:
assuming this is a cheap clone cuz the thing we're cloning only stores the channels

Reply:
yes only clones a sender